.circleci/config.yml

```diff
@@ -337,42 +348,67 @@ jobs:
     docker:
      - image: circleci/python:3.6-node-browsers
     steps:
-#      - checkout
       - checkout-creme
+      - install-py-dev-env:
+          python: "python3.6"
+      - create-creme-project
+      - setup-creme-statics
       - install-node-env
-#      - run: make eslint
-      - run: make -C << pipeline.parameters.source_path >> eslint
+      - run: cp << pipeline.parameters.source_path >>/.eslint* << pipeline.parameters.instance_directory >>
+      - run:
+          name: Javascript linting
+          working_directory: << pipeline.parameters.instance_directory >>
+          command: |
+              find << pipeline.parameters.source_path >>/creme -iname *.js | xargs --no-run-if-empty \
+                  node_modules/.bin/eslint \
+                  --no-eslintrc \
+                  --config .eslintrc \
+                  --ignore-path .eslintignore \
+                  --format stylish
+      - run:
+          name: Template javascript linting
+          working_directory: << pipeline.parameters.instance_directory >>
+          command: |
+              find << pipeline.parameters.source_path >>/creme -iname *.html | xargs --no-run-if-empty \
+                  node_modules/.bin/eslint \
+                  --no-eslintrc \
+                  --config .eslintrc \
+                  --ignore-path .eslintignore \
+                  --plugin template \
+                  --rule 'template/no-template-branch: 2' \
+                  --global '____' \
+                  --format stylish
 
   javascript-tests:
     docker:
      - image: circleci/python:3.6-node-browsers
     steps:
-#      - checkout
       - checkout-creme
       - install-creme-system-packages
       - install-py-dev-env:
           python: "python3.6"
-#      - run: cp ~/project/.circleci/circleci_settings.py creme/project_settings.py
-#      - run: python creme/manage.py generatemedia
       - create-creme-project
+      - setup-creme-statics
       - install-node-env
-      # TODO: fix (problems with static paths media hard-coded (see .karma.conf.js)
-      - run: make karma-ci
```

Comment:
I presume you can comment the rule in Makefile (remove my "TODO" comment) + an internal API breaking in the changelog.